web/themes/custom/votegov/templates/component/to-top.html.twig

```diff
@@ -0,0 +1,2 @@
+{{ attach_library('votegov/to_top') }}
+<a href="#" class="back-to-top"></a>
```

Comment:
I think we need to either add text in the link or a title attribute for this. Can you check with Amy.